stdlib/public/TensorFlow/Ops.swift

```diff
@@ -904,13 +904,13 @@ public extension Tensor where Scalar : Numeric & Comparable {
   /// Returns the index of the maximum value of the flattened scalars.
   @_inlineable @inline(__always)
   func argmax() -> Int32 {
-    return self.flattened().argmax(squeezingAxis: 0).scalarized()
+    return _TFGetScalarOrDie(flattened().argmax(squeezingAxis: 0).handle)
```

Comment:
should we add some tests?

Reply:
This patch is simply restoring the old graph generated by these ops. Tests will be added separately.